Limit `::-webkit-media-controls` to `audio` and `video` in `mediaControls.css`

Ahmad-S792 opened the pull request:
#### 7250ca3fb789c3b62649ace163bee10ad4d6de22
<pre>
Limit `::-webkit-media-controls` to `audio` and `video` in `mediaControls.css`

<a href="https://bugs.webkit.org/show_bug.cgi?id=270948">https://bugs.webkit.org/show_bug.cgi?id=270948</a>

Reviewed by Antti Koivisto.

This patch is to limit `-webkit-media-controls` selector to only
`audio` and `video` elements as they are only relevant elements.

* Source/WebCore/css/mediaControls.css:

Canonical link: <a href="https://commits.webkit.org/279491@main">https://commits.webkit.org/279491@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d1f9eddf9b5207e9a6b70507da377e77b6641acb

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/53631 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/32991 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/6143 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/56911 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/4357 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/55934 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/40471 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/4189 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/43454 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/2842 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/55728 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/31191 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/46363 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/24589 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/28037 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/3679 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/2512 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/49773 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/3860 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/58506 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/28793 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/3901 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/50860 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/29993 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/46518 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/50201 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/11690 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/30924 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/29770 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebCore/css/mediaControls.css:
```diff
@@ -27,7 +27,8 @@
 /* media controls */
-::-webkit-media-controls {
+audio::-webkit-media-controls,
+video::-webkit-media-controls {
```
`:is(audio, video)::-webkit-media-controls` also works, although I don't see much point in changing this codepath in general as I believe it's unused? cc @graouts @jernoble

Ahmad-S792 replied:
I opted for this based on @anttijk feedback on another bug's [comment](https://bugs.webkit.org/show_bug.cgi?id=268481#c2) for another pseudo-element, so I just thought to restrict this and also noticed that Blink also do it.

graouts replied:
I can confirm this codepath is unused aside from testing.

Ahmad-S792 replied:
I was checking today to see - if we can remove but it seems `Playstation` port list this in their [PlatformPlayStation.cmake](https://searchfox.org/wubkat/source/Source/WebCore/PlatformPlayStation.cmake) and [Adwaita.cmake](https://searchfox.org/wubkat/source/Source/WebCore/platform/Adwaita.cmake#30). So I think meanwhile it would be good to land. I will ask them separately, if we can remove this.

Commit:
Limit `::-webkit-media-controls` to `audio` and `video` in `mediaControls.css`

https://bugs.webkit.org/show_bug.cgi?id=270948

Reviewed by Antti Koivisto.

This patch is to limit `-webkit-media-controls` selector to only
`audio` and `video` elements as they are only relevant elements.

* Source/WebCore/css/mediaControls.css:

Canonical link: https://commits.webkit.org/279491@main
